core/src/components/cat-select/cat-select.tsx

```diff
@@ -21,6 +21,7 @@ import {
 } from 'rxjs';
 import { CatFormHint } from '../cat-form-hint/cat-form-hint';
 import { catI18nRegistry as i18n } from '../cat-i18n/cat-i18n-registry';
+import isEqual from 'lodash/isEqual';
```

Comment:
I would really like to keep this project lodash free. Is there any option to implement this without lodash. We are trying to get rid of lodash in the core project for various reasons and my fear is that once it is introduced here, we will use it more and more in this project

Reply:
Hmm okey, I will remove it